app/views/shared/_progress_navigation_buttons.html.erb

```diff
@@ -1,5 +1,6 @@
 <%# previous button params %>
 <% dont_show_prev_button = local_assigns[:dont_show_prev_button] ? dont_show_prev_button : false %>
+<% previous_link = local_assigns[:previous_link] ? previous_link : '#' %>
```

Comment:
is this needed? looks like it isn't being used in the href for the back button

Reply:
fixing